### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by vaishnavirshah**
## Issue that this pull request solves

 Closes: #99 

## Proposed changes

I have added an image for login page, changed the position of the login section and made it similar to Contact us Page.

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by vaishnavirshah** [comment]
@Kajol-Kumari  is this fine?
image

File: frontend/src/components/auth/loginPage-VaishnaviShah/login.css [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    left: -370px;
+    top: 300px;
+    height: 400px;
+    margin-bottom: 100px;
+    margin-top: -50px;
+  }
+}
+
+
+  /* .login-card {
```
Comment (mohit355) [inline frontend/src/components/auth/loginPage-VaishnaviShah/login.css:120]: Remove unnecessary code

Comment (vaishnavirshah) [reply frontend/src/components/auth/loginPage-VaishnaviShah/login.css]: Done @mohit355

**Review by mohit355 — APPROVED** [review]

**Review by salil-naik — CHANGES_REQUESTED** [review]
@vaishnavirshah Good job with the login page. I have a few suggestions. 

Please use **[bootstrap grid](https://getbootstrap.com/docs/4.0/layout/grid/)** to place image and login form beside each other. Using *position*, *translate* and *margin* properties to place elements in line is a bad approach. That layout can easily be achieved with the help of a bootstrap grid.  

Let us know if you need any help.

**Comment by vaishnavirshah** [comment]
> @vaishnavirshah Good job with the login page. I have a few suggestions.
> 
> Please use **[bootstrap grid](https://getbootstrap.com/docs/4.0/layout/grid/)** to place image and login form beside each other. Using _position_, _translate_ and _margin_ properties to place elements in line is a bad approach. That layout can easily be achieved with the help of a bootstrap grid.
> 
> Let us know if you need any help.

@salil-naik  I had tried using bootstrap initially but I was not able to import it into my login file, if you could help me with that - it will be great.

**Comment by salil-naik** [comment]
> > @vaishnavirshah Good job with the login page. I have a few suggestions.
> > Please use **[bootstrap grid](https://getbootstrap.com/docs/4.0/layout/grid/)** to place image and login form beside each other. Using _position_, _translate_ and _margin_ properties to place elements in line is a bad approach. That layout can easily be achieved with the help of a bootstrap grid.
> > Let us know if you need any help.
> 
> @salil-naik I had tried using bootstrap initially but I was not able to import it into my login file, if you could help me with that - it will be great.

You need not import anything. Directly use the bootstrap classes and it should work.

**Comment by vaishnavirshah** [comment]
@salil-naik I have used bootstrap grid in my code now. 
This is how it looks, is it fine?

image

And on smaller screen - 

image


Should I update the files?

**Comment by salil-naik** [comment]
@vaishnavirshah It looks better 👌 But you need to push your changes in order for me to check your code.

**Comment by vaishnavirshah** [comment]
@salil-naik I am not able to push my files into GitHub, could you help me with that?

image

**Review by Kajol-Kumari — APPROVED** [review]
@vaishnavirshah Thanks for incorporating the suggested changes
LGTM :+1:

**Pull request merged**